docs/content/backup_restore.md

```diff
@@ -32,10 +32,10 @@ console db-tools:backup --connection other_connection_name
 
 ### Excluded tables
 
-You may have configured [tables to be excluded in the bundle configuration](./configuration#excluded-tables).
+You may have configured [tables to be exclude in the bundle configuration](./configuration#excluded-tables).
```

Comment:
```suggestion
You may have configured [tables to exclude in the bundle configuration](./configuration#excluded-tables).
```

Reply:
Not my PR !

Reply:
Sorry, I forgot to rebase actual main branch. Fixed.